book.rb

```diff
@@ -0,0 +1,16 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Please make the rentals accessible e.g attr_accessor :rentals so that you can set or assign values to the rentals array 👍